src/main/java/ru/yandex/practicum/filmorate/controller/FilmController.java

```diff
@@ -0,0 +1,61 @@
+package ru.yandex.practicum.filmorate.controller;
+
+import lombok.Setter;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.web.bind.annotation.*;
+
+import ru.yandex.practicum.filmorate.controller.validation.FilmValidator;
+import ru.yandex.practicum.filmorate.controller.validation.ValidationException;
+import ru.yandex.practicum.filmorate.model.Film;
+
+import javax.validation.Valid;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+@Slf4j
+@RestController
+@RequestMapping("/films")
+public class FilmController {
+    private int id = 0;
+    @Setter
+    private boolean customValidation = false;
```

Comment:
Лучше переименовать в 
```suggestion
    private boolean isValid = false;
```
Так код будет проще читать.

Reply:
boolean customValidation используется только для тестов ручной валидации, поэтому в контексте других замечаний это поле удалено из классов